modules/gapi/perf/common/gapi_video_perf_tests_inl.hpp

```diff
@@ -154,6 +154,79 @@ PERF_TEST_P_(BuildPyr_CalcOptFlow_PipelinePerfTest, TestPerformance)
 
 //------------------------------------------------------------------------------
 
+#ifdef HAVE_OPENCV_VIDEO
+
+PERF_TEST_P_(BackgroundSubtractorPerfTest, TestPerformance)
+{
+    namespace gvideo = cv::gapi::video;
+    initTestDataPath();
+
+    gvideo::BackgroundSubtractorType opType;
+    bool detectShadows = false;
+    double learningRate = -1.;
+    std::string filePath = "";
+    std::size_t testNumFrames = 0;
+    cv::GCompileArgs compileArgs;
+    CompareMats cmpF;
+
+    std::tie(opType, detectShadows, learningRate, filePath, testNumFrames,
+             compileArgs, cmpF) = GetParam();
+
+    int histLength = 500;
+    double thr = -1;
+    if (opType == gvideo::TYPE_BS_MOG2)
+        thr = 16.;
+    else if (opType == gvideo::TYPE_BS_KNN)
+        thr = 400.;
+    gvideo::BackgroundSubtractorParams bsp(opType, histLength, thr, detectShadows, learningRate);
```

Comment:
Can you change name of the function? Is this name frozen? bsp is strange for me.

Reply:
Maybe I didn't understand correctly.. `bsp` is a variable, the type of it is `BackgroundSubtractorParams`, so the name is just after the type

Reply:
Object *. I thought that it is namespace gvideo::BackgroundSubtractorParams:: . My bad.
bsp is strange for me.
Ok.